ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/updates/TransactionRejections.scala

```diff
@@ -34,13 +24,11 @@ import scala.jdk.CollectionConverters._
   */
 private[kvutils] object TransactionRejections {
 
-  @nowarn("msg=deprecated")
```

Comment:
I think this deprecation warning had nothing to do with self service error codes. Same comment applies throughout this file

Reply:
AFAIK these annotations are checked by the compiler, so an invalid configuration (unnecessary annotation) would lead to a compilation failure. I cannot restore them and not break the build, they are redundant.